code/modules/cm_preds/yaut_weapons.dm

```diff
@@ -127,7 +144,7 @@
 	item_state = "scim"
 	attack_speed = 5
 	attack_verb = list("sliced", "slashed", "jabbed", "torn", "gored")
-	force = MELEE_FORCE_TIER_5
+	force = MELEE_FORCE_TIER_6
```

Comment:
the alt skin of scims isnt inheriting the new buff. while we're at it, just delete the redundant code if its the same as it would inherit from parent